### PR TITLE
id:6504 comment: Contact Us page - add Newsletter sign-up module

### PR DESCRIPTION
**Pull request by Guntrisoft**
- adding option to display the newsletter signup straight away
  and not use the Newsletter.js script to place it

### CODE REVIEW

File: app/views/shared/_news_signup_test.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="news-signup-test" data-dough-component="Newsletter">
+<div class="news-signup-test <% if defined?(visible) %>news-signup-test--with-image news-signup-test--in-article <% end %>" <% unless defined?(visible) %>data-dough-component="Newsletter"<% end %>>
```
Comment (tomas-stefano) [inline app/views/shared/_news_signup_test.html.erb]: Can you change from `defined?(visible)` to `local_assigns.has_key?(:visible)`?

**Comment by tomas-stefano** [comment]
:+1:

File: app/views/shared/_news_signup_test.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="news-signup-test" data-dough-component="Newsletter">
+<div class="news-signup-test <% if local_assigns.has_key?(:visible) %>news-signup-test--with-image news-signup-test--in-article <% end %>" <% unless local_assigns.has_key?(:visible) %>data-dough-component="Newsletter"<% end %>>
```
Comment (asmega) [inline app/views/shared/_news_signup_test.html.erb]: use a ternary here

Comment (asmega) [inline app/views/shared/_news_signup_test.html.erb]: actually use a ternary and move the double quote outside of the conditional. how does that sound?

Comment (asmega) [inline app/views/shared/_news_signup_test.html.erb]: oh wait they're different attributes. my fault :+1:

**Comment by asmega** [comment]
:+1:

**Pull request merged**